Cargo.toml

```diff
@@ -19,9 +19,9 @@ license = "MIT/Apache-2.0"
 travis-ci = { repository = "amethyst/amethyst", branch = "master" }
 
 [features]
-default = ["animation", "audio", "locale", "network", "renderer"]
```

Comment:
Did you mean to commit this?

Reply:
Oops should only exclude audio because of rust-windowing/winit#1255

Reply:
Oh. The hightlight on github looked like I removed all defaults. Removing audio was intended because of the issue.